src/view.js

```diff
@@ -402,17 +402,17 @@ ModelService.prototype.openBuffer = function(buffer, identifier, callback) {
 }
 
 function Int64(data) {
-    this.data = data;
+    this.data = String.fromCharCode(parseInt(data, 2));		// byte data converted to string
```

Comment:
Since this is losing information converting from int64 byte[] to Number probably shouldn't be done in the construct. It's more of a stop gap that could be applied in toString() until real code is available that supports > 2^53 -1.